static/js/play.js

```diff
@@ -49,6 +49,11 @@ let app = new Vue({
 
 
     mounted: async function() {
+        if (!("username" in serverData)) {
```

Comment:
This will prevent any logged in user from playing prompts, we only want this behavior for rated prompts. After looking at this though I realize that the redirect wasn't working due to a bug, so you don't have to worry about this anymore. I'll just revert it for you and then merge this.